power_grid_model_c/power_grid_model/include/power_grid_model/main_core/update.hpp

```diff
@@ -7,6 +7,7 @@
 #define POWER_GRID_MODEL_MAIN_CORE_UPDATE_HPP
 
 #include "state.hpp"
+#include <ranges>
 
 #include "../all_components.hpp"
```

Comment:
agreement within the team (following LLVM recommendations in https://llvm.org/docs/CodingStandards.html#include-style): STL libraries should always be included last, e.g.:

```suggestion
#include "state.hpp"

#include "../all_components.hpp"

#include <ranges>
```

Reply:
Swapped in the newest commit